### DISCUSSION TITLE
Optimize me #2, Piezo Crystals

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#15205] [Ideas and suggestions]
Most of their update time are those status effects "always", are you sure you need them?

https://github.com/user-attachments/assets/83216339-d88b-44e8-8901-984545c42e20

![Screenshot_8](https://github.com/user-attachments/assets/7e16d67d-e790-41c6-85bd-99207cb39f31)

Also, is it possible to desync their strikes?

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/pull/6042 (PR in our private repo, not yet merged to the unstable build)

**Reply by Regalis11** [reply]
Merged now, coming up in the next unstable build.
